data-formats/src/constants/mod.rs

```diff
@@ -426,6 +426,7 @@ pub enum ErrorCode {
 #[non_exhaustive]
 pub enum MfgStringType {
     SerialNumber = 0,
+    SerialNumberExt = 1,
```

Comment:
We can do a bit of bikeshedding here about naming (which is hard anyway but still). This new format provide a structured device identification so _maybe_ we could name it `Structured` or `fill_in_your_suggestion` - the reason for not using `SerialNumberExt` is because it doesn't really tell what `Ext` is and can be just as weird to then look at it and see mac addresses.

Reply:
I have two suggestions , that can give a bit more clarity on what these two types do: 
`pub enum MfgStringType {
                  MfgProvidedSerialNumber,
                  CustomSerialNumberWithMAC }`
or CustomSerialNumberWithMAC -> StructuredSerialNumberWithMAC

Reply:
First one, the current, we can't change w/o losing backwards compatibility. The second one as you suggested Irene is tricky as we might add other things than serial number and mac since it's just a key=value structure. uhm

Reply:
right.. how about , SerialNumberExt -> DeviceInfo or DeviceInfoStructure ? As you mentioned this field could be MAC address or any other info of the device which we might support later.
If we need a bit more specific related to device identification then it could be something like : 
DeviceIdInfo .
Also this will relate to the Field is same as we use in device credential & ov to represent this info .

Reply:
how about `SerialNumberExt` -> `MacDerivedSerialNumber`?

Reply:
The issue I have with mac and serial number in the new schema is that it won't really be tied to just those two (if any new requirements comes in the future, say people want the hostname?) - how about `StructuredDeviceInfo`?

Reply:
StructuredDeviceInfo sounds good to me, we can go ahead with that if everyone agrees !

Reply:
Great ,thanks all!